app/resources/view.html.template

```diff
@@ -265,7 +264,7 @@
 
             {% if linked_person_info %}
               <div class="other section">
-                <h3>{% trans "Possible duplicates" %}</h3>
+                <h2>{% trans "Possible Duplicates" %}</h2>
```

Comment:
I noticed that you are trying to change all the titles to title cased. What was the motivation?